gdcdictionary/schemas/diagnosis.yaml

```diff
@@ -18012,13 +18067,18 @@ properties:
         - Ovary, NOS
         - Ovary, Right
         - Pancreas
+        - Parametrium, left
+        - Parametrium, right
         - Parotid gland
         - Pelvic Peritoneum
+        - Pelvic lymph node(s)
         - Pelvis
+        - Peri-aortic lymph node(s)
```

Comment:
this is "Peri-aortic Lymph Node(s)" in jira

Reply:
Updated to what's in JIRA.